.github/workflows/main.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
 
   test_reconfig:
     runs-on: ubuntu-latest
-    timeout-minutes: 40
+    timeout-minutes: 50
```

Comment:
are you sure this is needed?  I'm seeing a successful run of this take ~20min, why bump to 50?

Reply:
test_reconfig also timed out a few times (https://github.com/faucetsdn/forch/actions/runs/305295427, https://github.com/faucetsdn/forch/actions/runs/305122643).